src/java/components/gpiomanager/src/main/java/hu/bme/mit/inf/modes3/components/gpiomanager/Main.java

```diff
@@ -13,13 +13,16 @@
  */
 public class Main {
 
-    /**
+	/**
      * @param args the command line arguments
      * @throws java.lang.Exception
      */
     public static void main(String[] args) throws Exception {
 
         GpioManager.loadGpioMappingFromFile(args[0]);
+        
+        GpioManager.setGpioWriter(new GpioWriter());
```

Comment:
If the comments above for the WRITER field are resolved, then please also replace it accordingly.